docs/api/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Does this directory need its own package.json? Can we manage the scripts in the `docs` root file instead?

Reply:
Deleted it and it didnt affect local